packet/models_tinkerbell.go

```diff
@@ -0,0 +1,160 @@
+package packet
```

Comment:
We have models.go, models_cacher.go, models_tinkerbell.go. As we discussed, it will nice to have file header comments to briefly describe which is for what.